code/harry_potter_store/src/store.py

```diff
@@ -0,0 +1,33 @@
+class Store:
+    book_types = ["1", "2", "3", "4", "5"]
+
+    def __init__(self, book_price=8):
+        self.book_price = book_price
+        pass
+
+    def _clean(self, to_buy_books: dict) -> dict:
+        return dict(filter(lambda item: item[1] > 0 and item[0] in Store.book_types, to_buy_books.items()))
+
+    def _decrement(self, to_buy_books: dict) -> dict:
+        return self._clean(dict(map(lambda i: [i[0], i[1] - 1], to_buy_books.items())))
+
+    # book_types = {"str_type": int_count}
```

Comment:
no, it is describe of dict in argument

Reply:
If it's for type hints - why not use standard [type annotations](https://docs.python.org/3/library/typing.html#typing.Dict) like `Dict[str, int]`?

Or if you want human-readable documentation, then use standard [Docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) and write it in human-readable way (like "values are amount of books of the given type...." instead of "int_count")

Reply:
done